binheap.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+class Heap(object):
+    """Create a Heap."""
+
+    def __init__(self, iterable=None):
+        """Initialize Heap."""
+        # if iterable:
+        #     self.container = list(iterable)
+        #     for i in self.container:
+        #         self.push(i)
+        # else:
```

Comment:
Why did you remove this? It's actually part of the assignment description: "The constructor for your heap should default to creating an empty heap, but allow for creating a populated given an iterable as an input."